plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -130,11 +123,7 @@ public void stop(boolean force) {
     if (devServer != null && (!force || serverState != IServer.STATE_STOPPING)) {
       setServerState(IServer.STATE_STOPPING);
       StopConfiguration.Builder builder = StopConfiguration.builder();
-      if (isDevAppServer1()) {
-        builder.adminPort(serverPort);        
-      } else {
-        builder.adminPort(adminPort);
-      }
+      builder.adminPort(serverPort);
```

Comment:
should appengine-plugins-core simply remove this method and use the server port?

Reply:
appengine-plugins-core removed admin port recently: https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/780/files#diff-cde49634f93d555a4ee296fe77c7294bL60

We will be able to remove this code too when we upgrade appengine-plugins-core once more.